modules/ecs/cluster/main.tf

```diff
@@ -26,15 +26,15 @@ resource "aws_security_group" "lb_security_group" {
     from_port = 80
     to_port   = 80
 
-    cidr_blocks = [element(var.whitelist, count.index)]
+    cidr_blocks = var.whitelist
   }
 
   ingress {
     protocol  = "tcp"
     from_port = 443
     to_port   = 443
 
-    cidr_blocks = [element(var.whitelist, count.index)]
+    cidr_blocks = var.whitelist
```

Comment:
oh while I'm thinking about naming things. Maybe var.whitelist can be allowed_ip_list or something like that 🤷

Reply:
I was thinking about that too. 😄 Went with `allow_list` since I'm using that in `nerves_hub_infrastructure` (local branch currently).